source/configuration/program-detail-page.md

```diff
@@ -110,22 +110,22 @@ The Program Detail Page is divided into a Left Hand Panel and a Right Hand Panel
 
 ### Configuring the Left Hand Panel
 The left panel can display a maximum of **6** attributes as label:value pairs. If you add more than 6 attributes, **Bento will display only the first 6 attributes without a warning or error message**.
-1. Open `$(src)/bento-frontend/src/bento/programDetailData.js`.
+1. Open `$(src)/packages/bento-frontend/src/bento/programDetailData.js`.
 2. Under `leftPanel`:
-   * Add an object ({dataField: ; label: })to the `attribute` list.
+   * Add an object ({dataField: ; label: })to the `attributes` list.
     * Set the field `dataField` to the GraphQL API query that returns the data field of your choice. This will be displayed as the value of in a label:value pair.
```

Comment:
Remove "your", replace with Set the field dataField to the GraphQL API query that returns the desired data field.